<?xml version="1.0"?>

<launch>

    <node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" name="teleop_key" output="screen">
        <remap from="cmd_vel" to="/key_vel" />
        <rosparam>
            vx_max: 0.6
            wz_max: 1.0
        </rosparam>
    </node>

</launch>
